packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -784,6 +822,48 @@ export function TransferPanelMain({
         </NetworkListboxPlusBalancesContainer>
       </NetworkContainer>
 
+      {isSmartContractWallet && (
+        <div className="mt-6">
+          <button
+            onClick={() => setShowAdvancedSettings(!showAdvancedSettings)}
+            className="flex flex-row items-center"
+          >
+            <span className=" text-lg">Advanced Settings</span>
+            {showAdvancedSettings ? (
+              <ChevronUpIcon className="ml-1 h-4 w-4" />
+            ) : (
+              <ChevronDownIcon className="ml-1 h-4 w-4" />
+            )}
+          </button>
+          {showAdvancedSettings && (
```

Comment:
as per above: force user to provide destination address whenever L1 or L2 are SCs (not just as an advanced settings feature)